doc/source/ray-core/fault_tolerance/actors.rst

```diff
@@ -1,8 +1,13 @@
-.. _actor-fault-tolerance:
-
-Fault Tolerance
+Actor tolerance
 ===============
 
+Similar to objects, there are two kinds of failures in actors, 1) the actor
+process dies; 2) the owner of the actor dies.
+
```

Comment:
Similar to objects, there are two kinds of failures in actors: 1) the actor
process dies and 2) the owner of the actor dies.